src/mon/OSDMonitor.cc

```diff
@@ -6156,7 +6157,8 @@ bool OSDMonitor::preprocess_command(MonOpRequestRef op)
       {"dedup_chunk_algorithm", DEDUP_CHUNK_ALGORITHM},
       {"dedup_cdc_chunk_size", DEDUP_CDC_CHUNK_SIZE},
       {"bulk", BULK},
-      {"read_ratio", READ_RATIO}
+      {"read_ratio", READ_RATIO},
+      {"data_layout_reformatting", DATA_LAYOUT_REFORMATTING},
```

Comment:
This is missing corresponding get and set entries in MonCommands.h (easy to miss!)